.github/workflows/mac_release.yml

```diff
@@ -13,13 +13,9 @@ jobs:
   mac_release:
     runs-on: macos-10.15
 
-    strategy:
-      matrix:
-        osx_target: ["10.13", "10.15"]
-
```

Comment:
Indeed, with this PR we are going to just be building 10.15 packages, at least on Github Actions.  This doesn't limit us to never make a 10.13 package, but the 10.13 packages coming out of Github Actions 10.15 machines was flawed.  A developer with a 10.13 machine (or 10.14) could certainly build a full release package and we can test it and post it with our list of installers.